staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4134,6 +4134,8 @@ type ServiceAccount struct {
 	// can be mounted in the pod, but ImagePullSecrets are only accessed by the kubelet.
 	// More info: https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod
 	// +optional
+	// +patchMergeKey=name
+	// +patchStrategy=merge
 	ImagePullSecrets []LocalObjectReference `json:"imagePullSecrets,omitempty" protobuf:"bytes,3,rep,name=imagePullSecrets"`
```

Comment:
Also need `patchStrategy:"merge" patchMergeKey:"name"` in the field tags (siblings to the json and protobuf tags) ?